### PR TITLE
update MAIN files for 'Report Bug' links for sle 15 sp4

### PR DESCRIPTION
**Pull request by cjschroder**
update MAIN files for 'Report Bug' links for sle 15 sp4

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by cwickert** [comment]
You forgot SLE Micro (5.2), I just did it. I also took the liberty to squash your two separate commits (guides and articles) into one, thus the forced-push.

We should also consider cherry-picking the changes to the `update-entities-sle15sp4` branch so we can deal with everything on one PR.

**Comment by sknorr** [comment]
Ok, I squashed both of your commits down into one. Then I noticed that the product does not exist in Bugzilla yet. Adding the ON-HOLD label.

File: xml/MAIN.SLEDS.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP3</dm:product>
+    <dm:product>PUBLIC SUSE Linux Enterprise Server 15 SP4</dm:product>
     <dm:assignee>fs@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/MAIN.SLEDS.xml:48]:
```suggestion
    <dm:assignee>taroth@suse.com</dm:assignee>
```

File: xml/art_pci-dss.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>SUSE Linux Enterprise Server 15 SP2</dm:product>
+    <dm:product>SUSE Linux Enterprise Server 15 SP4</dm:product>
     <dm:assignee>fs@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/art_pci-dss.xml:28]:
```suggestion
    <dm:assignee>taroth@suse.com</dm:assignee>
```

File: xml/art_raspberry-pi.xml [hunk 3]
```diff
@@ -28,7 +28,7 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>SUSE Linux Enterprise Server 15 SP3</dm:product>
+    <dm:product>SUSE Linux Enterprise Server 15 SP4</dm:product>
     <dm:assignee>fs@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/art_raspberry-pi.xml:32]:
```suggestion
    <dm:assignee>taroth@suse.com</dm:assignee>
```

File: xml/MAIN.SLEDS.xml [hunk 4]
```diff
@@ -60,7 +60,7 @@
    <dm:bugtracker>
     <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
     <dm:component>Documentation</dm:component>
-    <dm:product>PUBLIC SUSE Linux Enterprise Desktop 15 SP3</dm:product>
+    <dm:product>PUBLIC SUSE Linux Enterprise Desktop 15 SP4</dm:product>
     <dm:assignee>fs@suse.com</dm:assignee>
```
Comment (taroth21) [inline xml/MAIN.SLEDS.xml:64]:
```suggestion
    <dm:assignee>taroth@suse.com</dm:assignee>
```

**Review by taroth21 — CHANGES_REQUESTED** [review]
@cjschroder : Many thanks! I changed the Bugzilla assignees as defined [here](https://confluence.suse.com/display/SUSELinuxEnterpriseServer15SP4/%5BWIP%5D+Bugzilla+components+list). I also checked with Vincent when the 'PUBLIC' products will be available in Bugzilla for SP4.

**Review by taroth21 — APPROVED** [review]
Hi Carla, thanks! Vincent just pinged me and let me know that the 'PUBLIC' entries for SLES, SLED (and SLE-HA) 15 SP4 in Bugzilla have been created and are active now. However, SLE Micro 5.3 is not available in BZ yet (as it also has a different schedule from the rest)

**Comment by cjschroder** [comment]
Ok everyone, thanks for help, but this is a mess. Please do not make changes to my PRs unless I ask. Suggestions are better, and much appreciated.

**Pull request merged**